javabasics/src/main/java/javabasics/Library.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * This Java source file was generated by the Gradle 'init' task.
+ */
+package javabasics;
+
+import java.util.Random;
+
+
+public class Library {
+    public boolean someLibraryMethod() {
+        return true;
+    }
+
+    public int[] roll(int n){
+
+        int[] rollResult = new int[n];
+
+        for (int i = 0; i < n; i++) {
+
+            Random generator = new Random();
+            Float randomNumber = generator.nextFloat();
+
+            //This feels like the wrong way to get a number between 1 and 6, but it works, so...
```

Comment:
It might look a little different, but there's many ways to generate random numbers. It is a subject unto itself in computer science. What's important is that you are generating within the range you want. 👍